### DISCUSSION TITLE
Empty Chat title

### DISCUSSION THREAD

**Discussion by divyam234** [danny-avila/LibreChat#4182] [Troubleshooting]
### What happened?

Chat  title gets empty after page reload it should auto generate some text without setting it manually.

### Steps to Reproduce

Reload page after new chat

### Screenshots

![image](https://github.com/user-attachments/assets/6454aef1-0146-4b41-908d-a72981b85160)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Can't reproduce, no relevant logs or endpoint config provided, see my video:
https://github.com/user-attachments/assets/45c4ddf1-cbe3-4e0b-9143-eea0d34f74e4
